lib/cylc/task_proxy.py

```diff
@@ -466,8 +466,11 @@ def command_log(self, ctx):
         job_log_dir = self.get_job_log_dir(
             self.tdef.name, self.point, submit_num, self.suite_name)
         job_activity_log = os.path.join(job_log_dir, "job-activity.log")
-        with open(job_activity_log, "ab") as handle:
-            handle.write(ctx_str)
+        try:
+            with open(job_activity_log, "ab") as handle:
+                handle.write(ctx_str)
+        except IOError:
+            pass
```

Comment:
Can we write the string to suite log instead, with a warning about the job-activity log write failure?